src/neuvector/zarf.yaml

```diff
@@ -33,9 +33,9 @@ components:
         version: 0.1.0
         localPath: chart
     images:
-      - docker.io/neuvector/controller:5.2.2-s1
-      - docker.io/neuvector/manager:5.2.2-s1
+      - docker.io/neuvector/controller:5.2.3
+      - docker.io/neuvector/manager:5.2.3
```

Comment:
Just noting for anyone reviewing - updated the images and chart version here because the `-s1` prefix was not being handled properly by renovate (and is not present on future chart/image versions).

Reply:
TBH I'm surprised this didn't work when the -distroless and the 1.19.3-tetrate-fips-v0 tags were working

Reply:
Yeah I was confused as well but it didn't seem to pick up the newer version until I dropped the `-s1` so I just went with it. It looks like all future (and previous?) chart versions don't have that suffix so I think it makes sense to just drop it.